### ISSUE TITLE
Code needs to be cleaned up more

### ISSUE THREAD

**Issue by JacksonElia** [NCSU-High-Powered-Rocketry-Club/AirbrakesV2#108]
Honestly our code is pretty good at this point, but it could always be better. We need to:
- [x] Clean up `constants.py`
- [ ] Add comments to the tests (oh boy)
- [ ] Do another go at changing var names and function names to be clearer and convey more info
- [ ] Flesh out `scripts/` a little more, maybe add a nicer UI for them or something
- [ ] Expand the readme, maybe add a little more documentation/guidance where to start for new members

**Comment by harshil21** [comment]
Duplicate of #38

**Comment by wlsanderson** [comment]
> Duplicate of #38

so real

**Comment by harshil21** [comment]
> Flesh out scripts/ a little more, maybe add a nicer UI for them or something

This can probably be done along with/after #106.
